kani-driver/src/list/output.rs

```diff
@@ -176,5 +118,42 @@ pub fn json(
 
     serde_json::to_writer_pretty(writer, &json_obj)?;
 
+    println!("Wrote list results to {}", std::fs::canonicalize(JSON_FILENAME)?.display());
+
     Ok(())
 }
+
+/// Output the results of the list subcommand.
+pub fn output_list_results(
```

Comment:
Nit: Can you please move this function to the top? This is the entry point of this module, so it's easier to read the code from it.